### PR TITLE
Update Gauss_Easter_Date.cpp

### PR DESCRIPTION
**Pull request by M4rkoHR**
Algorithm for determining the easter date for a given year

## Assigned Issue Number ?
#525 

## Language Opted ?
C++

## Number of files added or updated ?
1

## Name of these files ?
Algorithm => Dynamic Programming => Gauss_Easter_Date => Gauss_Easter_Date.cpp

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've referred the correct issue number.
- [x] I've fill up this entire template correctly.

### CODE REVIEW

File: Algorithms/Dynamic_Programming/Gauss_Easter_Date/Gauss_Easter_Date.cpp [hunk 1]
```diff
@@ -1,49 +1,44 @@
...
-	}
-	cout << easterD << endl;
+	gaussAlgorithm(year, &easterMonth, &easterDay); //calculate month and day
+	cout << "Date of easter for year " << year << " is: " << ((easterMonth == 3) ? "March " : "April ") << easterDay << endl;
 	char ch;
 	cin >> ch; //wait for user input
 	return 0;
 }
+
+void gaussAlgorithm(int year, int *month, int *day) //https://math.stackexchange.com/questions/896954/decoding-gauss-easter-algorithm
```
Comment (M4rkoHR) [inline Algorithms/Dynamic_Programming/Gauss_Easter_Date/Gauss_Easter_Date.cpp:19]: Directly implemented from the link in the comment

Comment (ahampriyanshu) [reply Algorithms/Dynamic_Programming/Gauss_Easter_Date/Gauss_Easter_Date.cpp:19]: Great :+1:

**Review by ahampriyanshu — APPROVED** [review]
Perfect Code :+1: 
Thanks @M4rkoHR
Happy Contributing :smile: :tada:

**Comment by ahampriyanshu** [comment]
con
Major compare conflict :warning: 
Can't merge :no_entry: 
Sorry :disappointed:

**Pull request closed without merging**